tsnet/tsnet_test.go

```diff
@@ -947,16 +942,136 @@ func sendData(logf func(format string, args ...any), ctx context.Context, bytesC
 	return nil
 }
 
-func TestUserMetrics(t *testing.T) {
+func TestUserMetricsByteCounters(t *testing.T) {
 	flakytest.Mark(t, "https://github.com/tailscale/tailscale/issues/13420")
```

Comment:
Should we remove the `flakytest` if we expect this to fix the flakes?
(same for the `TestUserMetricsRouteGauges` below).